staticconf/config.py

```diff
@@ -439,6 +439,31 @@ def has_changed(self):
         return last_mtime < self.last_max_mtime
 
 
+def create_LoggingMTimeComparator(err_logger):
+    """Return an MTimeComparator subclass that calls err_logger(filename)
+    if os.path.getmtime(filename) fails. err_logger is called within the
+    context of the raised OSError, thus sys.exc_info() can be used to
+    examine the exception."""
```

Comment:
Using `compare_func` instead of `err_logger` significantly increases the work required to move a service over to the new behavior, which we would like to minimize. I have moved the logging logic inside of MTimeComparator, however (see comment below).